tm2py/controller.py

```diff
@@ -112,15 +125,25 @@ def _init_emme_manager(self):
 
     def run(self):
         """Main interface to run model"""
-        self._iteration = None
-        self.validate_inputs()
-        for iteration, name, component in self._queued_components:
-            if self._iteration != iteration:
-                self.logger.log_time(f"Start iteration {iteration}")
-            self._iteration = iteration
-            self._component = component
-            component.run()
-            self.completed_components.append((iteration, name, component))
+        with self.logger:
```

Comment:
I"m not sure I love having to use the context?  Is there a reason for it?  cc @lmz who may also have thoughts here.

Reply:
I'll defer to @lmz  - generally I don't love having to indent everything that I want to log

Reply:
(granted I realize that you can do it on the calling function)

Reply:
Sorry I'm behind on this... 
I don't like having to indent everything that I want to log either.

Reply:
I'll create a separate issue